dsl/security.go

```diff
@@ -519,6 +519,8 @@ func TokenField(tag interface{}, name string, args ...interface{}) {
 // Scope has two uses: in JWTSecurity or OAuth2Security it defines a scope
 // supported by the scheme. In Security it lists required scopes.
 //
+// Scope can be defined in BasicAuth and APIKeyAuth for custom usage.
```

Comment:
Please just add these two expressions to the list below (`Scope must appear in …`).

Reply:
Changed ;)